### PR TITLE
ESM Engage Page Update

### PR DESCRIPTION
**Pull request by Khutch25**
## Done

Here's the updated copy, layout for ESM Engage Page 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

### CODE REVIEW

File: templates/engage/14-04-esm.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        <p>We can work with your team to start planning for Ubuntu 14.04 End of Life and recommend a support package that best suits your needs. </p>
+        <p><a href="/support/contact-us?product=server-esm" class="p-button--neutral js-invoke-modal">Get in touch</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-shallow" id="register-section">
+    <div class="row u-equal-height">
+      <div class="jsBrightTALKEmbedWrapper col-12" style="width:100%; height:100%; position:relative"><script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : 6793, "language": "en-US", "commId" : 349275, "displayMode" : "standalone", "height" : "auto" }</script><script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script></div>
+        </div>
```
Comment (anthonydillon) [inline templates/engage/14-04-esm.html]: Remove extra closing div

File: templates/engage/14-04-esm.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+          <span class="mktInput mktLblRight">
+            <input class="mktFormCheckbox" name="canonicalUpdatesOptIn" id="canonicalUpdatesOptIn" value="yes" type="checkbox">
+            <label class="contextual-footer__text--label" for="canonicalUpdatesOptIn">I agree to receive information about Canonical’s products and services.</label>&nbsp;<span class="mktFormMsg"></span>
+          </span>
+        </li>
+        <li class="p-list__item u-no-margin--top">
+          <p>In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/newsletter">Canonical’s Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</p>
+        </li>
+        <li class="p-list__item">
+          <span class="u-off-screen"><input type="text" name="_marketo_comments" value=""></span>
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/engage/14-04-esm.html]: You can add `aria-label` as an attr here to solve the issue.

File: templates/engage/14-04-esm.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    <p>Canonical is offering Extended Security Maintenance (ESM) to provide ongoing security fixes and essential packages for Ubuntu LTS releases.</p>
+    <p><a href="/esm" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'Support - ESM', 'eventLabel' : 'Find out more about ESM', 'eventValue' : undefined });">Find out more about ESM</a></p>
+  </div>
+   
+        
+      </div>
+    </div>
+  </div>
+
+<!--
```
Comment (anthonydillon) [inline templates/engage/14-04-esm.html]: Remove commented out code

**Comment by anasereijo** [comment]
- [ ] remove the shield icon (we already have a security icon on the hero section)

- [ ] hide the padlock on mobile

Everything else is looking good 👍

**Comment by anasereijo** [comment]
👍 @anthonydillon

**Pull request merged**